### ISSUE TITLE
Enhancement: Footer Accessibility - Social media icons lack descriptive text. and colour

### ISSUE THREAD

**Issue by mrosevere** [mrosevere/hackathon-elemental-game#96]
Issue: Social media icons lack descriptive text.

Recommendation:

html
<a href="https://www.facebook.com" aria-label="Visit our Facebook page">
    <i class="fab fa-facebook fa-2x" aria-hidden="true"></i>
</a>
